.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    'no-proto': 0
```

Comment:
The strict and max-len rules have been removed. If these rules were intentionally removed, ignore this comment. Otherwise, consider adding 'strict: 0' and 'max-len: 0' back to the rules object.

Reply:
I figured out how to fix the bug so I added it back